AustLII and NZLII.js

```diff
@@ -1,121 +1,229 @@
 {
 	"translatorID": "5ed5ab01-899f-4a3b-a74c-290fb2a1c9a4",
 	"label": "AustLII and NZLII",
-	"creator": "Bill McKinney and Sebastian Karcher",
-	"target": "^https?://www\\.(austlii\\.edu\\.au|nzlii\\.org)",
-	"minVersion": "2.1.9",
+	"creator": "Justin Warren, Philipp Zumstein",
+	"target": "^https?://www\\d?\\.(austlii\\.edu\\.au|nzlii\\.org)",
```

Comment:
Do we have these numbers also for the `nzlii` part? Otherwise we might move it into the bracket to the `austlii` part only.